aicsimageio/readers/lif_reader.py

```diff
@@ -147,13 +147,13 @@ def _get_image_data(
             # Create the fill array shape
             # Drop the YX as we will be pulling the individual YX planes
             retrieve_shape: List[int] = []
-            use_selected_or_np_map: Dict[str, Optional[int]] = {}
+            use_selected_or_np_map: Dict[str, int] = {}
             for dim, index_op in zip(retrieve_dims, retrieve_indices):
                 if dim not in [DimensionNames.SpatialY, DimensionNames.SpatialX]:
                     # Handle slices
                     if index_op is None:
                         # Store the dim for later to inform to use the np index
-                        use_selected_or_np_map[dim] = None
+                        use_selected_or_np_map[dim] = -1
```

Comment:
this is a good candidate for a symbolic constant, like `MISSING_DIM_SENTINEL_VALUE` or whatever

Reply:
Agree.